setup.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+from os import path
+from setuptools import setup, find_packages
+
+this_directory = path.abspath(path.dirname(__file__))
+with open(path.join(this_directory, 'README.md'), encoding='utf-8') as f:
```

Comment:
nice 😎

Reply:
yeah, kinda. would be nice if it was magic instead though :stuck_out_tongue_closed_eyes: